server/src/models/events.model.ts

```diff
@@ -0,0 +1,17 @@
+import "reflect-metadata";
+import { Model, Table, Column, DataType } from "sequelize-typescript";
+
+@Table({ tableName: "events" })
+export class Event extends Model {
+  @Column({
+    type: DataType.STRING,
+    allowNull: false,
+  })
+  eventName!: string;
```

Comment:
why are you using ! here?

Reply:
This is to accurately type the data and eliminate nulls and undefineds from the table. I think that's a good proposition. The guides do suggest using this, but it's not a requirement to push the project forward.